### PR TITLE
Feat/pilot 988

### PR DESCRIPTION
**Pull request by JiayuZhang15**
## Summary

(Summarize the changes)

## JIRA Issues

(What JIRA issues this merge request is related to)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [ ] Yes
- [x] No

## Test Directions

(Additional instructions for how to run tests or validate functionality if not covered by unit tests)

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ services:
       - CONFIG_CENTER_ENABLED=false
       - RDS_DB_URI=postgresql://postgres:postgres@db:5432/approval_db
       - RDS_SCHEMA_DEFAULT=test_schema
-      - NEO4J_SERVICE=http://127.0.0.1
```
Comment (gregmccoy) [inline docker-compose.yaml:14]: Can you add PROJECT_SERVICE

File: app/routers/v1/api_copy_request/api_copy_request.py [hunk 2]
```diff
@@ -331,7 +331,13 @@ def complete_request(self, project_code: str, data: PUTRequest):
 
         submitted_at = request_obj.submitted_at.strftime('%Y-%m-%d %H:%M:%S')
         completed_at = request_obj.completed_at.strftime('%Y-%m-%d %H:%M:%S')
-        notify_user(request_obj.submitted_by, data.username, project_code, submitted_at, completed_at)
+        await notify_user(
```
Comment (gregmccoy) [inline app/routers/v1/api_copy_request/api_copy_request.py:334]: I'm seeing multiple instances where you broke code into multiple lines when it didn't need to be as we allow line length up to 120. Is your flake8 or precommit not set to 120?

File: docker-compose.yaml (hunk 1 above)
Comment (JiayuZhang15) [reply docker-compose.yaml:14]: Actually the environment variables are no longer needed for newer versions of common packages. I could remove them all.

File: app/routers/v1/api_copy_request/api_copy_request.py (hunk 2 above)
Comment (JiayuZhang15) [reply app/routers/v1/api_copy_request/api_copy_request.py:334]: Nope, mine was 80, could update to 120.

File: docker-compose.yaml (hunk 1 above)
Comment (gregmccoy) [reply docker-compose.yaml:14]: yes my mistake

File: app/routers/v1/api_copy_request/api_copy_request.py (hunk 2 above)
Comment (gregmccoy) [reply app/routers/v1/api_copy_request/api_copy_request.py:334]: Yes please update it to 120 for the future, it's fine for now

**Review by gregmccoy — APPROVED** [review]

**Pull request merged**